sender/sql.go

```diff
@@ -107,7 +108,7 @@ func (sq *SQL) prep() {
 		} else {
 			sq.list = append(sq.list, dbElement{data, element})
 		}
-		if sq.Driver == "mysql" {
+		if sq.Driver == "mysql" || sq.Driver == "sqlite3" {
```

Comment:
I think this should accept "sqlite" as well as "sqlite3", as they're pretty much synonyms (as far as I know).

Reply:
I can't agree on this one. There is a big difference between version 2 and 3 of sqlite despite the fact that version 2 is old the use of sqlite as a general name could be a bad choice in case where version 2 would have to be used.

Reply:
I hope nobody runs sqlite2, its last release seems to be from 2005. But the configuration check is very helpful and will guide the user to the correct term, so I am fine with keeping it as-is.

Example error:
```
FATA[0000] Failed to configure Skogul                    category=cmd cmd=main error="configuration for sender `sqlite' doesn't verify: unsuported database driver sqlite - must be `mysql` or `postgres` or `sqlite3`"
```